packages/code-studio/vite.config.ts

```diff
@@ -101,32 +101,6 @@ export default defineConfig(({ mode }) => {
       emptyOutDir: true,
       sourcemap: true,
       target: 'esnext',
-      rollupOptions: {
-        output: {
-          manualChunks: id => {
```

Comment:
Curious what changed so that we don't need this anymore?

Reply:
I think part of the reason for chunking like this initially was we wanted to split out monaco and plotly, but it didn't actually do any lazy loading. I'm not sure if the manual chunks works properly if I specify just those that we want

We added mathjax to parse markdown latex math syntax which was a heavy package, so that is part of the driver for splitting the chunks. I can check if things properly lazy load (mathjax and plotly) w/ the manual chunks and just adding a mathjax chunk

Reply:
I honestly don't have any understanding of what the original config was for or whether it matters, just wanted to document reasoning that it used to be required but now is not.

Reply:
Found out Vite still splits chunks out properly w/ lazy loading even if I use manual chunks for the node_modules stuff. This allowed me to write an e2e test verifying that plotly and mathjax chunks don't load. Previously, the Vite chunks could be random. Like the mathjax chunk ended up in 1 file called `svg-abc123.js`

This might be slightly less efficient for chunking since the smaller dependencies like rehype-math get included in the general vendor chunk than the auto chunking, but I like that it can be tested more easily